jina/clients/__init__.py

```diff
@@ -28,33 +28,33 @@ async def _get_results(self, *args, **kwargs):
         if self.args.return_results:
             return result
 
-    @deprecated_alias(buffer=('input_fn', 1), callback=('on_done', 1), output_fn=('on_done', 1))
-    def train(self, input_fn: InputFnType,
+    @deprecated_alias(buffer=('inputs', 1), callback=('on_done', 1), output_fn=('on_done', 1))
```

Comment:
Should we add two deprecated aliases here? One to buffer from before, and now one to `input_fn`?

Reply:
yes I think so

Reply:
Yes, it will become 
`@deprecated_alias(buffer=('inputs', 1), input_fn=('inputs', 0), callback=('on_done', 1), output_fn=('on_done', 1))`
where 0 means warning and the old one is still supported

Reply:
check